mgmtworker/mgmtworker/task_consumers/wait_for_replication_mixin.py

```diff
@@ -0,0 +1,57 @@
+########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+############
+
+import logging
+
+from cloudify_agent.worker import (
+    CloudifyOperationConsumer,
+    CloudifyWorkflowConsumer
+)
+try:
+    from cloudify_premium import mgmtworker_utils
+except ImportError:
+    mgmtworker_utils = None
+
+logger = logging.getLogger('mgmtworker')
+
+
+class WaitForReplicationMixin(object):
```

Comment:
not really needed now? or anyway, with the current premium-side impl

Reply:
I guess not, so I'll just remove the whole thing